trillian/ctfe/handlers.go

```diff
@@ -288,7 +283,7 @@ func (li *logInfo) Handlers(prefix string) PathHandlers {
 		prefix + ct.GetEntryAndProofPath:  AppHandler{Info: li, Handler: getEntryAndProof, Name: GetEntryAndProofName, Method: http.MethodGet},
 	}
 	// Override mirror methods.
```

Comment:
I think best not use "override" in the comment as Go doesn't have this concept. And "methods" isn't really right, for HTTP the method is Get or Post.

How about "Remove endpoints not provided by mirrors".

Reply:
Done.